### PR TITLE
Ft home page

### PR DESCRIPTION
**Pull request by felixodette**
In this project we:
- Add linter configuration locally and ignore node modules.
- Create a HTML file.
- Create a CSS file.
- Create a test.md file and exclude it from git.
- Write a good README file based on the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
...
+
+Setup
+
+Install
+
+Usage
+
+Run tests
+
+Deployment
```
Comment (Kingstalux) [inline README.md]: - [ ] Kindly edit the empty sections with information about your project or simply remove all the necessary sections from your README file in order to make it more professional 👍

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @felixodette , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f644550476b6e5269456e6965632f67697068792e676966


## Highlights
✅ Linters are passing
✅ Good commit messages

## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly provide a pull request description explaining the changes implemented in the pull request 😄 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by felixodette** [comment]
Hi @Kingstalux, thanks for the quick review. I have made changes to:
- the pull request description, and
- the README file.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (ShadyShawkat) [inline .gitignore:1]: - [ ] [OPTIONAL] Kindly add `node_modules` to your `.gitignore` file

**Review by ShadyShawkat — APPROVED** [review]
## Hi @felixodette, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**